docs/resources.html

```diff
@@ -214,25 +220,36 @@ <h2 id="improving-the-quality-of-reviews">Improving the quality of reviews</h2>
+				<p><a href="https://aom.org/research/journals/discoveries" target="_blank">Academy of Management Discoveries</a> has implemented “<a href="https://aom.org/events/event-detail/2024/01/19/higher-logic-calendar/discoveries_through_prose" target="_blank">Discoveries Through Prose</a>”, where authors are invited to present their empirical findings in prose rather than in more traditional “academese”. This can include (but is not limited to) continuous anecdotes or guiding examples, placing methods/thick descriptions in appendices, no traditional academic headings, and incorporating data/findings throughout. See <a href="https://doi.org/10.5465/amd.2021.0068" target="_blank">Kahn (2021)</a> for an example manuscript of this type.</p>
+				<h3 id="publishing-scientific-criticism">Publishing scientific criticism</h2>
+				<p>In this context, “scientific criticism” means a formal written response to a published finding. Journals can handle scientific criticism in different ways, e.g. publishing letters, commentaries, or online comments (post-publication peer review, PPPR). <a href="https://doi.org/10.31222/osf.io/4w3kb" target="_blank">Hardwicke et al. (2022)</a> assessed PPPR policies and practices at journals and found that most journals don’t currently offer options for submitting PPPR, and those that do tend to impose strict length/timing limits. They also found that even at journals that do offer PPPR, very few PPPRs actually end up being published. Hardwicke and colleagues make several recommendations for journals that are interested in adopting PPPR in <a href="https://doi.org/10.31222/osf.io/4w3kb" target="_blank" title="Hardwicke et al., 2022">Box 2 of their article</a>.</p>
+				<h2 id="peer-review-innovations">Peer review innovations</h2>
+				<p>Peer review is undergoing a sustained and multifaceted debate about different attempts to improve its perceived effectiveness, objectivity, transparency, and long-term sustainability. Many of these have already been discussed extensively above, e.g. deanonymizing reviews and/or reviewers, acknowledging and incentivizing reviewing, etc. There are many more – for an analytical overview of ongoing peer review innovations see <a href="http://www.doi.org/10.31235/osf.io/8hdxu" target="_blank">Kaltenbrunner et al. (2022)</a>. They categorize peer review innovations as either being about the <i>object</i> of peer review (what is being peer reviewed), the <i>aim</i> of peer review (why is peer review being performed), the <i>role</i> of peer review <i>actors</i> (who performs peer review), the <i>nature</i> of peer review (how is peer review performed), or the <i>openness/transparency</i> of peer review (what is available to whom during and after peer review).</p>
```

Comment:
I always do this "wrong" myself, but semantic markup like `<em>` and `<strong>` is preferred to `<i>` and `<b>`

Reply:
ooh ok, thanks!